index.bs

```diff
@@ -651,3 +779,81 @@ The [=remote end steps=] are:
 
 2. Return [=success=] with data |body|
 
+### Subscribe ### {#command-session-subscribe}
+
+The <dfn export for=commands>subscribe</dfn> command enables certain events either
+globally or for a set of browsing contexts
+
+Issue: This needs to be generalised to work with realms too
+
+<dl>
+   <dt>Command Type</dt>
+   <dd>
+      ```
+      SubscribeCommand = {
+        method: "session.subscribe",
+        params: SubscribeParameters
+      }
+
+      SubscribeParameters = {
+        events: [text],
+        contexts?: [BrowsingContext],
```

Comment:
```suggestion
        targets?: [BrowsingContext],
```

I presume this is where we want to end up? To always give different target types separately seems like it would be tedious, but it does depend on what the client code looks like.

Reply:
Yeah, probably but I'd rather refactor later once we can actually address realms.